src/getItems.js

```diff
@@ -1,34 +1,31 @@
-const leaderboardItems = [
-  {
-    id: 1,
-    name: 'Man City',
-    score: '9',
-  },
-  {
-    id: 2,
-    name: 'PSG',
-    score: '8',
-  },
-  {
-    id: 3,
-    name: 'Liverpool',
-    score: '12',
-  },
-  {
-    id: 4,
-    name: 'Real Madrid',
-    score: '9',
-  },
-  {
-    id: 5,
-    name: 'Inter',
-    score: '7',
-  },
-  {
-    id: 6,
-    name: 'Man United',
-    score: '7',
-  },
-];
+const getLeaderboardItems = async () => {
+  let leaderboardItems;
 
-export default leaderboardItems;
\ No newline at end of file
+  const list = document.getElementById('leaderboard-list');
+  list.innerHTML = '';
+
+  try {
+    const leaderboardURL = await fetch('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/gKX6ORGkPPBMcNp9S7oe/scores', {
+      method: 'GET',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+    });
+    leaderboardItems = await leaderboardURL.json();
+  } catch (err) {
+    console.log('No result found');
+  }
+
+  if (leaderboardItems.result.length) {
+    leaderboardItems.result.forEach((item) => {
+      list.innerHTML += `
+      <td>${item.user}:</td>
+      <td>${item.score}</td>
+      `;
+    });
+  } else {
+    list.innerHTML += '<h4>No result found</h4>';
+  }
+};
```

Comment:
[OPTIONAL] I think you could make sure that once we've got the results from the API, the HTML from the list is clean. This way we will make sure that before appending new results our document is empty. Otherwise, users that might spam the refresh button will get repeated results.

![image](https://user-images.githubusercontent.com/79114233/142345180-4921ff0a-a87b-4938-ae25-4a6b9dc5893b.png)